tests/test_butler.py

```diff
@@ -283,19 +283,35 @@ class PosixDatastoreButlerTestCase(ButlerTests, lsst.utils.tests.TestCase):
     configFile = os.path.join(TESTDIR, "config/basic/butler.yaml")
     fullConfigKey = "datastore.formatters"
 
+    def testStringification(self):
```

Comment:
To fit in more with the style of these subclasses, please define `testStringification` once in the baseclass and define the parameters here. So in this case:
```python
datastoreStr = "datastore='butler_test_repository'"
registryStr = "registry='sqlite:///:memory'"
```

Reply:
These all make sense, and the branch has been updated with them.